### DISCUSSION TITLE
Installing UTM on iOS and iPadOS 17.5 in the EU

### DISCUSSION THREAD

**Discussion by econcz** [utmapp/UTM#6357] [UTM Help (iOS)] [answered]
Good day! I have a general question. Since iOS and iPadOS 17.5 it is now possible to install apps regularly (without sideloading) from websites in the EU (or if one creates an EU-based AppleID). Could you please launch such an installation website for UTM?

In my opinion, this website would be a good alternative to jailbreaking and prevent JIT enabling issues caused by sideloading.

Thanks!

**Comment by yihuanlin** [comment]
They also allow emulators on App Store idk if that means UTM can be submitted to the Store

**Comment by osy** [comment, marked as answer]
https://twitter.com/utmapp/status/1791099417866727667
